syntax = "proto3";
import "common.proto";
package autodrive.workflow.proto;

message ModuleTaskMsg {
  // Header
  MsgHeader header = 1;
  // Error
  Error error = 2;
  // DAG id
  uint32 dag_id = 3;
  // module id
  uint32 module_id = 4;
  // loop start time
  uint64 start_time = 5;
  // loop end time
  uint64 end_time = 6;
}